android/src/app/organicmaps/car/util/RoutingUtils.java

```diff
@@ -0,0 +1,253 @@
+    builder.addStep(createNextStep(context, info), createTravelEstimate(info.distToTurn, 0));
+    mCachedTrip = builder.build();
+    return mCachedTrip;
+  }
+
+  @NonNull
+  public static Distance createDistance(@NonNull final app.organicmaps.util.Distance distance)
+  {
+    int displayUnit;
+    switch (distance.mUnits)
+    {
+    case Kilometers:
+      displayUnit = distance.mDistance >= 10.0 ? Distance.UNIT_KILOMETERS : Distance.UNIT_KILOMETERS_P1;
+      break;
+    case Feet:
+      displayUnit = Distance.UNIT_FEET;
+      break;
+    case Miles:
+      displayUnit = distance.mDistance >= 10.0 ? Distance.UNIT_MILES : Distance.UNIT_MILES_P1;
+      break;
+    case Meters:
+    default:
+      displayUnit = Distance.UNIT_METERS;
+      break;
+    }
+
+    return Distance.create(distance.mDistance, displayUnit);
+  }
+
+  @NonNull
+  public static LaneDirection createLaneDirection(@NonNull SingleLaneInfo.LaneWay laneWay, boolean isRecommended)
+  {
+    int shape = LaneDirection.SHAPE_UNKNOWN;
+    switch (laneWay)
+    {
+    case REVERSE:
+      shape = LaneDirection.SHAPE_U_TURN_LEFT;
+      break;
+    case SHARP_LEFT:
+      shape = LaneDirection.SHAPE_SHARP_LEFT;
+      break;
+    case LEFT:
+      shape = LaneDirection.SHAPE_NORMAL_LEFT;
+      break;
+    case SLIGHT_LEFT:
+    case MERGE_TO_LEFT:
+      shape = LaneDirection.SHAPE_SLIGHT_LEFT;
+      break;
+    case SLIGHT_RIGHT:
+    case MERGE_TO_RIGHT:
+      shape = LaneDirection.SHAPE_SLIGHT_RIGHT;
+      break;
+    case THROUGH:
+      shape = LaneDirection.SHAPE_STRAIGHT;
+      break;
+    case RIGHT:
+      shape = LaneDirection.SHAPE_NORMAL_RIGHT;
+      break;
+    case SHARP_RIGHT:
+      shape = LaneDirection.SHAPE_SHARP_RIGHT;
+      break;
+    }
+
+    return LaneDirection.create(shape, isRecommended);
+  }
+
+  @NonNull
+  public static Maneuver createManeuver(@NonNull final CarContext context, @NonNull RoutingInfo.CarDirection carDirection, int roundaboutExitNum)
+  {
+    int maneuverType = Maneuver.TYPE_UNKNOWN;
+    switch (carDirection)
+    {
+    case NO_TURN:
+    case GO_STRAIGHT:
+      maneuverType = Maneuver.TYPE_STRAIGHT;
+      break;
+    case TURN_RIGHT:
+      maneuverType = Maneuver.TYPE_TURN_NORMAL_RIGHT;
+      break;
+    case TURN_SHARP_RIGHT:
+      maneuverType = Maneuver.TYPE_TURN_SHARP_RIGHT;
+      break;
+    case TURN_SLIGHT_RIGHT:
+      maneuverType = Maneuver.TYPE_TURN_SLIGHT_RIGHT;
+      break;
+    case TURN_LEFT:
+      maneuverType = Maneuver.TYPE_TURN_NORMAL_LEFT;
+      break;
+    case TURN_SHARP_LEFT:
+      maneuverType = Maneuver.TYPE_TURN_SHARP_LEFT;
+      break;
+    case TURN_SLIGHT_LEFT:
+      maneuverType = Maneuver.TYPE_TURN_SLIGHT_LEFT;
+      break;
+    case U_TURN_LEFT:
+      maneuverType = Maneuver.TYPE_U_TURN_LEFT;
+      break;
+    case U_TURN_RIGHT:
+      maneuverType = Maneuver.TYPE_U_TURN_RIGHT;
+      break;
+    // TODO (AndrewShkrob): add support for CW (clockwise) directions
```

Comment:
Basically, it's a general issue but it will be used only in AA. Maybe we can leave it as it is and check how will it work. 
If there is a way to determine the type of traffic (right/left-hand) then we can easily fix this TODO.
Is it possible?

Reply:
Trace the usage of data/countries_meta.txt